### ISSUE TITLE
NodeJS API - CreateProgram isn't able to find `tslint:recommened` set of rules

### ISSUE THREAD

**Issue by nikhilhassija** [palantir/tslint#4893]
### Bug Report

- __TSLint version__: 5.20.1
- __TypeScript version__: 3.7.2
- __Running TSLint via__: Node.js API

#### Reproduction using [TSLint Playground](https://palantir.github.io/tslint-playground)
Not possible, using as NodeJs API

#### TypeScript code being linted

```ts
var someVar = "Global var";

const someFunc = () => {
    var someVar = "Local var";

    console.log(someVar);
};

someFunc();
```

with `tslint.json` configuration:

```json
{
    "extends": "tslint:recommended",
    "rules": {}
}
```

running `tslint` using the following script:

```ts
let tslint = require("tslint");

const lint = () => {
    let linter = tslint.Linter.createProgram("./tslint.json");
};

lint();
```

#### Actual behavior
Fails with error `FatalError: error TS6053: File 'tslint:recommended' not found.`

#### Expected behavior
Should be able to parse the `tslint.json` configuration correctly and give lint errors. 

Running with the same config via CLI works as expected.

**Comment by nikhilhassija** [comment]
I've created a demo repo to reproduce this bug. 

https://github.com/nikhilhassija/tslint_test

Use `node linter.js` to run, it'll throw the error that it's not able to find `tslint:recommended`

**Comment by jplatte** [comment]
I'm getting this same error message using `node_modules/.bin/tslint`.

**Comment by jplatte** [comment]
Found the issue: You have to point `tslint` to the `tsconfig.json`, not `tslint.json`.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
